app/uk/gov/hmrc/economiccrimelevyregistration/controllers/NotableErrorController.scala

```diff
@@ -49,4 +53,14 @@ class NotableErrorController @Inject() (
       )
     )
   }
+
+  def orgAlreadyRegistered: Action[AnyContent] = authoriseWithoutEnrolmentCheck { implicit request =>
```

Comment:
I'm thinking maybe a better name for this would be `groupAlreadyEnrolled`, since `orgAlreadyRegistered` may clash with the scenario where we check after GRS with ETMP.